src/Spaces/dss_transform.jl

```diff
@@ -196,6 +196,13 @@ function weighted_dss_start! end
 function weighted_dss_internal! end
 function weighted_dss_ghost! end
 
+# for backward compatibility
+function weighted_dss2! end
+function weighted_dss_start2! end
+function weighted_dss_internal2! end
+function weighted_dss_ghost2! end
+function dss2! end
```

Comment:
where are these methods defined?

Reply:
These methods are defined in `src/Fields/Fields.jl`. These are just used for backward compatibility and will be eventually removed.

Reply:
But it looks like not all the same methods are defined as before?

Reply:
Currently, only dss functions defined in the fields are currently called from external packages. So, backward compatible definitions are only added for these functions. The remaining functions are only used internally!